src/tests/test_serialization.cpp

```diff
@@ -57,52 +57,52 @@ BOOST_AUTO_TEST_CASE( utils_split_test )
 
 	{
 		auto split = utils::split(test_string);
-		std::array<std::string, 7> expect = {"a", "bb", "ccc  ||  d", "ee", "fff  | |  g", "hh", "iii"};
+		std::array expect = {"a", "bb", "ccc  ||  d", "ee", "fff  | |  g", "hh", "iii"};
```

Comment:
This probably no longer compiles because of the type change. My suggestion here is to just append `s` to each constant. (Note: use of that literal suffix does require `using namespace std::literals;` if I recall correctly.)

Reply:
Other tests already did the same thing without the literal.

Reply:
Oh, I misread the code. Sorry about that. This one is fine.